src/android-native/src/main/java/io/liteglue/SQLitePlugin.java

```diff
@@ -496,34 +495,20 @@ private void closeDatabaseNow(String dbname) {
      * @param cbc - JS callback
      */
     private void attachDatabase(String dbName, String dbNameToAttach, String dbAlias, CallbackContext cbc) {
-        SQLiteDatabase currentDb = this.getDatabase(dbName);
-        SQLiteDatabase attachDb = this.getDatabase(dbNameToAttach);
-
-        if (attachDb == null || ! attachDb.isOpen()) {
-            if (cbc != null) cbc.error("Database to attach (" + dbNameToAttach + ") is not open");
-            return;
-        }
-
-        if (currentDb == null || ! currentDb.isOpen()) {
-            if (cbc != null) cbc.error("Database " + dbName + " is not open");
-            return;
-        }
-
-        File dbfile = this.getContext().getDatabasePath(dbNameToAttach);
-        String filePathToAttached = dbfile.getAbsolutePath();
-
-        String stmt = "ATTACH DATABASE '" + filePathToAttached + "' AS " + dbAlias;
-        try {
-            this.executeSqlStatementQuery( currentDb, stmt, new JSONArray(), cbc );
```

Comment:
It's incorrect to execute code right here.
We have DBRunner thread in charge of access to DB.